arangod/RestServer/DatabaseFeature.cpp

```diff
@@ -1616,21 +1570,14 @@ void DatabaseFeature::closeDroppedDatabases() {
 
   // Build the new value:
   auto oldList = _databasesLists.load();
-  decltype(oldList) newList = nullptr;
-  try {
-    newList = new DatabasesLists();
-    newList->_databases = _databasesLists.load()->_databases;
-  } catch (...) {
-    delete newList;
-    throw;
-  }
+  auto newList = _databasesLists.create();
+  newList->_databases = oldList->_databases;
```

Comment:
Maybe better to separate _databasesLists on two atomic shared_ptr.

In which case we don't needed to touch _databases here and in gc thread
and _droppedDatabases in close thread